i18n/en/docusaurus-plugin-content-docs/current/guias/transacciones.md

```diff
@@ -0,0 +1,65 @@
+---
+id: transacciones
+title: Prepare Transactions
+sidebar_label: Prepare Transactions
+---
+
+To execute actions in a LACChain EOSIO contract it is necessary to prepare and execute transactions, here you can observe some considerations about this process.
```

Comment:
To execute transaction on LACChain EOSIO it is necessary to include at least two actions for each transaction.

The smart contract action you would like to execute and and  `writer::run` action which includes the authorization of the writer node who will provide the necessary network resources to execute the transaction.

 here is an example transaction:

Reply:
@AngeloCG97  creo que podemos ampliar mucho mas la explicacion sobre este tema , alli te puse un ejemplo de un texto mas especifico.